contrib/externs/chrome_extensions.js

```diff
@@ -10262,3 +10262,116 @@ chrome.downloads.DeterminingFilenameEvent = function() {};
  * https://developer.chrome.com/extensions/downloads#event-onDeterminingFilename
  */
 chrome.downloads.onDeterminingFilename;
+
+
+/**
+ * @const
+ * @see https://developer.chrome.com/extensions/declarativeContent
+ */
+chrome.declarativeContent = {};
+
+
+/**
+ * @enum {string}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcherInstanceType
+ */
+chrome.declarativeContent.PageStateMatcherInstanceType = {
```

Comment:
Thanks for figuring out the right format for these `InstanceType` types, but I think we'd be better off dropping them.
Here's my reasoning.

1. Even though it is small having these definitions has a cost in machine parsing time and human maintenance.
2. AFAICT there is no API that requires these values or exposes them to the client of the API. They seem to be totally internal and of no use to the clients of the API.

Can you show me where I'm wrong about the second point?
If not, let's just dump these until the API changes in some way that makes them useful.
Thanks.

Reply:
FWIW, the internal reviewer is also asking to drop these.